fastlane/lib/fastlane/new_action.rb

```diff
@@ -11,7 +11,7 @@ def self.fetch_name
       puts "examples: 'testflight', 'upload_to_s3'".green
       name = ask('Name of your action: ')
       until name_valid?(name)
-        puts 'Name invalid!'
+        puts 'Name invalid, try again!'
```

Comment:
Could there be a breakdown as to why it's invalid so the user has an idea how to fix this.